packages/@aws-cdk/aws-codepipeline/lib/private/cross-region-support-stack.ts

```diff
@@ -53,11 +60,13 @@ export class CrossRegionSupportConstruct extends Construct {
     super(scope, id);
 
     const createKmsKey = props.createKmsKey ?? true;
+    const enableKeyRotation = props.enableKeyRotation ?? false;
 
     let encryptionAlias;
     if (createKmsKey) {
       const encryptionKey = new kms.Key(this, 'CrossRegionCodePipelineReplicationBucketEncryptionKey', {
         removalPolicy: cdk.RemovalPolicy.DESTROY,
+        enableKeyRotation: enableKeyRotation,
```

Comment:
Let's only pass this if `props.enableKeyRotation` is `true`. This will preserve the current templates if `props. enableKeyRotation`  is not passed. So:

```suggestion
        enableKeyRotation: props.enableKeyRotation === true ? true : undefined,
```